docs/Metrics-System.md

```diff
@@ -51,20 +51,36 @@ The following shows the details of the available metrics.
 * CapacityTotal: Total capacity of the file system in bytes.
 * CapacityUsed: Used capacity of the file system in bytes.
 * CapacityFree: Free capacity of the file system in bytes.
+* PathsTotal: Total number of files and directories in the file system.
 * UnderFsCapacityTotal: Total capacity of the under file system in bytes.
 * UnderFsCapacityUsed: Used capacity of the under file system in bytes.
 * UnderFsCapacityFree: Free capacity of the under file system in bytes.
 * Workers: Number of the workers.
-* FilesTotal: Total number of files and directories in the file system.
-* FilesCheckpointed: Total number of the files checkpointed.
-* FilesCreated: Total number of files and directories created.
-* FilesDeleted: Total number of files and directories deleted.
+* DirectoriesCreated: Total number of directories created.
```

Comment:
do you think it would be possible to create subsection for each group? otherwise, it is not clear that these are several logical blocks of metrics